README.md

```diff
@@ -100,6 +100,8 @@ PublicLab.Editor uses grunt - the javascript task runner for compilation of the
 
 Make changes to the files in the `/src/` directory, then run `grunt build` to compile into `/dist/PublicLab.Editor.js`. This will use `grunt-browserify` to concatenate and include any node modules named in `require()` statements. You'll then be able to try it out in `/examples/index.html`. Run `grunt` and leave it running to build as you go.
 
+You can also run `grunt debug` to have *grunt-browserify* to include Source Maps for easy debugging. This way you can locate the module from where the error is generating.
```

Comment:
```suggestion
You can also run `grunt debug` to have *grunt-browserify* to include Source Maps for easy debugging. This way you can locate the module from where the error is generating. This is for use in development only. 
```

Reply:
Hey @geekychaser , please merge the above suggestion by @jywarren . Thanks!